Iryna_Ramanets/Module_1_Java/Java_Fundamentals/HW2_T2/src/com/epam/cdp/lists/ListsPerformanceEvaluator.java

```diff
@@ -0,0 +1,96 @@
+package com.epam.cdp.lists;
+
+import java.util.ArrayList;
+import java.util.LinkedList;
+import java.util.List;
+import java.util.Random;
+
+public class ListsPerformanceEvaluator {
```

Comment:
I keep comments only in the file, but they are applicable for all classes for the task.
1. A lot of copy-paste code
2. I suppose you can have only one main method and just call appropriate class and method to get specific results